share/messages_to_clients.txt

```diff
@@ -9361,6 +9361,9 @@ ER_EXPLICIT_DECRYPTION_OF_ONLINE_ENCRYPTED_TABLESPACE
 ER_SEQUENCE_TABLE_SIZE_LIMIT
   eng "The number of records generated by SEQUENCE_TABLE() cannot exceed %llu (%llu requested). Try increasing @@tf_sequence_table_max_upper_bound to a larger value."
 
+ER_D_T_E_CHANGE_WHILE_ENCRYPTION_THREADS_NOT_ZERO
```

Comment:
IMHO, this message tries to explain the entire situation. Can we be precise and say ER_DTE_ENCRYPTION_THREADS_ACTIVE

Reply:
done